### PR TITLE
Chore: optimize the package dependencies

### PR DESCRIPTION
**Pull request by barnettZQG**
### Description of your changes

This PR is pre-change for #5487, removing the dependency on the apiserver package. I will move the apiserver and the concerned packages to [VelaUX](https://github.com/kubevela/velaux) repository.

I have:

- [ ] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: references/cli/addon.go [hunk 1]
```diff
@@ -231,18 +230,6 @@ func AdditionalEndpointPrinter(ctx context.Context, c common.Args, k8sClient cli
...
-			fmt.Printf("\nInitialized admin username and password: admin / %s \n\n", service.InitAdminPassword)
-		}
-		fmt.Println(`To open the dashboard directly by port-forward:`)
-		fmt.Println()
-		fmt.Println(`    vela port-forward -n vela-system addon-velaux 9082:80`)
-		fmt.Println()
-		fmt.Println(`Select "local | velaux | velaux" from the prompt.`)
-		fmt.Println()
-		fmt.Println(`Please refer to https://kubevela.io/docs/reference/addons/velaux for more VelaUX addon installation and visiting method.`)
-	}
```
[the remark this answers is not in the document]
Comment (barnettZQG) [reply references/cli/addon.go:245]: Yes

[the remark this answers is not in the document]
Comment (barnettZQG) [reply references/cli/addon.go:245]: Next version, we don't generate the default password.

[the remark this answers is not in the document]
Comment (wonderflow) [reply references/cli/addon.go:245]: Do you mean the first page is just going to the password setting page?

[the remark this answers is not in the document]
Comment (barnettZQG) [reply references/cli/addon.go:245]: Yeah

File: pkg/utils/app/app.go [hunk 2]
```diff
@@ -46,9 +45,8 @@ var ErrPublishVersionNotChange = errors.Errorf("the PublishVersion is not change
 var ErrRevisionNotChange = errors.Errorf("the revision is not changed")
 
 // RollbackApplicationWithRevision make the exist application rollback to specified revision.
-func RollbackApplicationWithRevision(ctx context.Context, cli client.Client, appName, appNamespace, revisionName, publishVersion string) (*v1beta1.ApplicationRevision, *v1beta1.Application, error) {
-
-	revisionCtx := apiutils.WithProject(ctx, "")
+// revisionCtx the context used to manage the application revision.
+func RollbackApplicationWithRevision(ctx, revisionCtx context.Context, cli client.Client, appName, appNamespace, revisionName, publishVersion string) (*v1beta1.ApplicationRevision, *v1beta1.Application, error) {
```
Comment (Somefive) [inline pkg/utils/app/app.go:49]: Recommend not to use double context here.

**Review by Somefive — APPROVED** [review]
Generally LGTM.

**Review by FogDong — APPROVED** [review]

**Review by wonderflow — APPROVED** [review]

**Pull request merged**